File: lua/test/exists_test.lua
-- DomainWhois SDK exists test

local sdk = require("domain-whois_sdk")

describe("DomainWhoisSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# DomainWhois SDK exists test

require "minitest/autorun"
require_relative "../DomainWhois_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = DomainWhoisSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
